plugin-base/src/test/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/bean/BinanceBeanV4Test.java

```diff
@@ -1891,4 +1891,26 @@ void testIgnoringTransferBetweeinMainAndFundingWallet() {
 
         assertEquals(expectedProblem.getParsingProblemType(), actual.getParsingProblems().get(0).getParsingProblemType());
     }
+
+    @Test
+    void testBNB_Fee_Deduction() {
```

Comment:
Please, rename the method to testBnbFeeDeducation

Reply:
fixed